src/data/protocols/index.ts

```diff
@@ -1,2 +1,8 @@
-export * from './add-guardian-repository'
-export * from './encrypter'
+export * from './cryptography/hash-comparer'
+export * from './cryptography/hash-generator'
+export * from './cryptography/token-generator'
+export * from './cryptography/token-decoder'
+export * from './guardian/add-guardian-repository'
+export * from './guardian/load-guardian-by-email-repository'
+export * from './guardian/load-guardian-by-id-repository'
+export * from './guardian/update-access-token-repository'
```

Comment:
The directory "protocols" should contain an "index.ts" file that exports the contents of its directories:

```typescript
export * from './cryptography'
export * from './guardian'
```